python/paddle/nn/__init__.py

```diff
@@ -23,136 +23,237 @@
 
 from . import initializer
```

Comment:
remove `from . import something` if not needed

Reply:
done